portfolio/src/main/webapp/index.html

```diff
@@ -1,31 +1,3 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Vincent Stevenson</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <h1>Vincent's Portfolio</h1>
-    <div id="links">
-        <a href="index2.html">Gaming</a>
-    </div>
-    <div id="content">
-      <img src=/images/vstevens.jpg>
-      </br>
-      <a href="https://www.linkedin.com/in/vincent-stevenson-189631181">Check out my LinkedIn!</a>
-      </br>
-      <a href="https://github.com/frostyclawz">Check out my GitHub!</a>
-      </br>
-      <p>Hi! My name is Vincent Stevenson and I am a STEP Intern during Summer 2020.</p>
-      <p>I was born in July of 2000 in Dallas, Texas. I live currently in Arlington, Texas.
-          I am an only child in my family. I currently study Mechanical Engineering and Physics
-          at Xavier Univerisity of Lousiana in New Orleans, Louisiana. 
-      </p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+<head>
+    <meta http-equiv="refresh" content=""
```

Comment:
It looks like something extra was removed here. Should content be populated/do we need to close this tag?